config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml

```diff
@@ -94,6 +94,7 @@ periodics:
 
               TIMESTAMP=$(date +%s)
               K8S_BUILD_VERSION=$(curl https://storage.googleapis.com/kubernetes-release-dev/ci/latest.txt)
+              jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > $ARTIFACTS/metadata.json
```

Comment:
It is not working!
```
╰─$ K8S_BUILD_VERSION=123 jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ cat metadata.json
{}
```

Reply:
I tested it and even checked with running test-pj.sh.. hmm will check again.

Reply:
[root@rajalakshmi-workspace test-infra]# export K8S_BUILD_VERSION=123
[root@rajalakshmi-workspace test-infra]# jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
[root@rajalakshmi-workspace test-infra]# cat metadata.json
{
  "k8s-build-version": "123"
}
[root@rajalakshmi-workspace test-infra]#

Reply:
It is working fine,

```shell
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ export K8S_BUILD_VERSION=1.2.3
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ jq --arg key0 'k8s-build-version' --arg value0 $K8S_BUILD_VERSION '. | .[$key0]=$value0' <<<'{}' > metadata.json
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$ cat metadata.json
{
  "k8s-build-version": "1.2.3"
}
╭─manjunath@Manjunaths-MacBook-Pro-2 ~
╰─$
```